mediaEvents.js

```diff
@@ -179,6 +179,11 @@ var MediaEvents = (function () {
                 type: 'segment',
                 premature: true
             });
+
+            // Reset state at end of video
+            this.lastKnownPlaybackTime = 0;
```

Comment:
hmm, but between the set and your reset, there's just the flush event, which doesn't pass this as a value? I can't see what the point is of having 2?